### DISCUSSION TITLE
Question on Youtube "si" url tracking parameter

### DISCUSSION THREAD

**Discussion by NearestWater262** [TeamNewPipe/NewPipe#10472] [Questions]
Sometimes people send me these links like

`youtu.be/videoid?si=sharetrackingid`

If i am to open this link in newpipe, will it automatically remove this from the link? Or will youtube servers be sent this at all or see it?

Thank you for looking.

**Comment by opusforlife2** [comment]
Newpipe discards everything except the video/playlist etc. ID. So no worries there.

**Reply by NearestWater262** [reply]
Thanks very much
